caravel/controllers/forms.py

```diff
@@ -33,11 +33,11 @@ class ListingForm(flask_wtf.Form):
     
 
 class NewListingForm(ListingForm):
-    principal = PrincipalField("Seller",
-        validators=[DataRequired(), OnlyValid()])
+    principal = PrincipalField("Seller", validators=[DataRequired()])
```

Comment:
I see two issue with that:
1. Would we reconcile the two? i.e., if someone uses Shibboleth to login as "foo@uchicago.edu" then later posts a listing as "foo@uchicago.edu" we need to reconcile them. 
2. Does not prevent a malicious attacker from impersonating a "uchicago.edu" email address. Or rather, it makes the question of authenticating users (person A is really person A) our problem instead of trying to abstract it out to Shibboleth.